cypress/integration/features/task/createDeleteTaskFromTaskPage.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Task - To create and delete tasks starting from task overview page by the teacher.
+
+  As a teacher I want to create and delete a new task on the task overview page so that the student can submit it
+
+ Scenario: As a user, I want to be able to create simple tasks without course
```

Comment:
instead of 'user' we can specify the the role --> As a teacher, I want to...?

Reply:
I changed it.